File: RadioControl/app/src/main/java/com/nikhilparanjape/radiocontrol/activities/SplashActivity.java
package com.nikhilparanjape.radiocontrol.activities;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.nikhilparanjape.radiocontrol.R;

/**
 * Created by Nikhil on 4/24/2016.
 */
public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        finish();
    }
}

File: RadioControl/app/src/main/java/com/nikhilparanjape/radiocontrol/activities/SplashActivity.kt
package com.nikhilparanjape.radiocontrol.activities

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.nikhilparanjape.radiocontrol.BuildConfig
import com.topjohnwu.superuser.Shell

/**
 * Created by Nikhil on 4/24/2016.
 *
 * @author Nikhil Paranjape
 *
 *
 */
class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }
}